charts/kargo/values.yaml

```diff
@@ -177,6 +177,13 @@ controller:
   ## @param controller.enabled Whether the controller is enabled.
   enabled: true
 
+  ## All settings relating to shared credentials (used across multiple kargo projects)
+  globalCredentials:
+    ## @param controller.globalCredentials.enable Whether to enable the controller to look for credentials stored in specific namespaces used to share credentials.
+    enabled: false
+    ## @param controller.namespaces List of namespaces to look for shared credentials as CSV (comma separated values).
+    namespaces: ""
```

Comment:
@lrotim, I think it would feel more natural for this to be a list of strings instead of a single, comma-delimited string.